File: src/test/samples/validSamples/validSample.swift
import Foundation

//#region FirstRegion
let x = 42
//#endregion

//  #region Second Region
class MyClass {
    // #region     InnerRegion
    func myMethod() {}
    //   #endregion   ends InnerRegion

    // #region     
    func myMethod2() {}
    // #endregion   
}
//   #endregion

File: src/test/samples/validSamples/validSample.go
package main

import "fmt"

//#region FirstRegion
var x = 42
//#endregion

// #region      Second Region
type MyClass struct{}

// #region InnerRegion
func (m MyClass) myMethod() {}
        //        #endregion ends InnerRegion

        // #region
func (m MyClass) myMethod2() {}
//#endregion

// #endregion

func main() {
    fmt.Println("Sample")
}
